crates/oxc_linter/src/rules/react/jsx_props_no_spread_multi.rs

```diff
@@ -0,0 +1,100 @@
+use std::collections::HashMap;
+
+use oxc_ast::{ast::JSXAttributeItem, AstKind};
+use oxc_diagnostics::OxcDiagnostic;
+use oxc_macros::declare_oxc_lint;
+use oxc_span::{Atom, Span};
+
+use crate::{context::LintContext, rule::Rule, AstNode};
+
+fn jsx_props_no_spread_multi_diagnostic(span0: Span, span1: Span) -> OxcDiagnostic {
```

Comment:
```suggestion
fn jsx_props_no_spread_multi_diagnostic(first_spread: Span, second_spread: Span, prop_name: &str) -> OxcDiagnostic {
```

Reply:
I’ve added prop_name as an argument.
https://github.com/oxc-project/oxc/pull/4866/commits/36f69a7a690d9af8f768619c2c7c1f06884eb118

FYI, the span now accepts a single Vec due to the following change:
https://github.com/oxc-project/oxc/pull/4866#discussion_r1716193012